packages/cursorless-everywhere-talon-e2e/src/quickjsTest.ts

```diff
@@ -0,0 +1,102 @@
+import type { ActionDescriptor, CommandLatest } from "@cursorless/common";
+import { activate } from "@cursorless/cursorless-everywhere-talon-core";
+import * as std from "std";
+import talonMock from "./talonMock";
+
+let hasFailed = false;
+
+async function runTests() {
+  await activate(talonMock, "test");
```

Comment:
I would prefer if we tested the real js and mocking talon imports rather than directly calling activate here, but I don't have time to think about whether there's a better way so I'll go with your instincts here

Reply:
I have no idea how I would be able to do that in node. This was the only solution I could come up with.

Reply:
For node I don't care. I mean for quickjs

Reply:
Should we have different entry points and build different bundles for node and for quick js then? I'm trying to keep it as simple as possible.